src/pages/Movies.vue

```diff
@@ -1,19 +1,42 @@
 <template>
   <div class="container" style="margin-top:15px">
     <div class="row" v-for="movie in movies" :key="movie.id">
-      <b-card
-        no-body
-        class="overflow-hidden offset-md-1"
-        style="max-width: 800px; margin-top:3px;cursor:pointer;"
-        @click="redirectToMovie(movie.id)"
-      >
+      <b-card no-body class="overflow-hidden offset-md-1" style="max-width: 800px; margin-top:3px;">
         <b-row no-gutters>
           <b-col md="4">
-            <b-card-img :src="movie.image_url" alt="Image" class="rounded-0"></b-card-img>
+            <b-card-img
+              :src="movie.image_url"
+              alt="Image"
+              class="rounded-0"
+              style="cursor:pointer"
+              @click="redirectToMovie(movie.id)"
+            ></b-card-img>
           </b-col>
           <b-col md="8">
             <b-card-body :title="movie.title">
-              <b-card-text>{{movie.description | truncate(300)}}</b-card-text>
+              <b-card-text>{{movie.description | truncate(200)}}</b-card-text>
+              <div class="form-inline">
```

Comment:
Da li si ovo mogao da imas komponentu za MovieReaction ili tako nesto? Pa da tamo imas ovaj html? Vidim da vec imas to u komponenti iznad? Zar nisu isti? Ako su isti onda nova komponenta